apps/aehttp/test/aehttp_hyperchains_SUITE.erl

```diff
@@ -718,6 +727,79 @@ epoch_with_slow_parent(Config) ->
                  [rpc(?NODE1, aec_chain_hc, epoch_start_height, [N]) || N <- lists:seq(1, EndEpoch)]),
     ok.
 
+%%%=============================================================================
+%%% Pinning
+%%%=============================================================================
+
+get_pin(Config) ->
+
+    %% note: we are actuall in epoch=2, but the pins are for the previous epoch
+    Repl1 = aecore_suite_utils:http_request(aecore_suite_utils:external_address(), get, "hyperchain/pin-tx", []),
+    {ok, 200, #{<<"epoch">> := 1}} = Repl1,
```

Comment:
check that `last` (or `height`in this code) is last of epoch 1, viz `?CHILD_EPOCH_LENGTH - 1`.

Reply:
Doesn't epoch 1 start on height=1 and end on height=10, not 0-9? i.e test viz just ?CHILD_EPOCH_LENGTH

Reply:
Sure looks like it when I test below...

Reply:
With check I mean, make sure the test case validates this. You can return anything in the http request. A test validates that the returned value is correct.